recipe/meta.yaml

```
@@ -24,25 +25,30 @@ requirements:
   run:
     - python
     - mccabe >=0.7.0,<0.8.0
-    - pycodestyle >=2.12.0,<2.13.0
-    - pyflakes >=3.2.0,<3.3.0
+    - pycodestyle >=2.14.0,<2.15.0
+    - pyflakes >=3.4.0,<3.5.0
 
 test:
+  source_files:
+    - tests
+    - bin/gen-pycodestyle-plugin
   imports:
     - flake8
     - flake8.api
     - flake8.formatting
     - flake8.main
     - flake8.options
     - flake8.plugins
-  requires:
-    - pip
   commands:
-    # Cannot run pytest since PyPI distributed source does not come with tests.
     - pip check
+    - flake8 --help
+    - pytest -v tests
+  requires:
+    - pip
+    - pytest
```

```suggestion
    - pytest !=3.0.5,!=5.2.3
```
https://github.com/PyCQA/flake8/blob/7.3.0/tox.ini#L7C11-L7C26